Runtime/Scripts/MeshSyncInstanceRenderer.cs

```diff
@@ -15,7 +15,9 @@ internal class MeshSyncInstanceRenderer : MonoBehaviour
 
         [HideInInspector]
         [SerializeField] private Matrix4x4[] transforms;
-        [SerializeField] private GameObject reference; 
+        [SerializeField] private GameObject reference;
+        [SerializeField] private GameObject instanceParent;
```

Comment:
Might simplify code / reduce .transform calls  if we store `Transform` instead of `GameObject`

Reply:
Done